elide-core/src/main/java/com/yahoo/elide/core/RequestScope.java

```diff
@@ -380,4 +380,11 @@ public void saveObjects() {
     public boolean useFilterExpressions() {
         return useFilterExpressions;
     }
+
+    public String getQueriedCollectionRelationName() {
```

Comment:
There are a couple of issues here:
1) A collection isn't always a 'relation'.  It could be a root collection.  The name of this function is misleading.
2) This function doesn't do the right thing if the path is not a collection but rather an element (/authors/1).